hosted/aks/support_matrix/support_matrix_importing_test.go

```diff
@@ -78,18 +78,18 @@ var _ = Describe("SupportMatrixImporting", func() {
 				})
 
 				By("checking service account token secret", func() {
-					success, err := clusters.CheckServiceAccountTokenSecret(ctx.RancherClient, clusterName)
+					success, err := clusters.CheckServiceAccountTokenSecret(ctx.RancherAdminClient, clusterName)
 					Expect(err).To(BeNil())
 					Expect(success).To(BeTrue())
 				})
 
 				By("checking all management nodes are ready", func() {
-					err := nodestat.AllManagementNodeReady(ctx.RancherClient, cluster.ID, helpers.Timeout)
+					err := nodestat.AllManagementNodeReady(ctx.RancherAdminClient, cluster.ID, helpers.Timeout)
 					Expect(err).To(BeNil())
 				})
 
 				By("checking all pods are ready", func() {
-					podErrors := pods.StatusPods(ctx.RancherClient, cluster.ID)
+					podErrors := pods.StatusPods(ctx.RancherAdminClient, cluster.ID)
```

Comment:
Looking at https://github.com/rancher/shepherd/blob/main/extensions/provisioning/verify.go#L174, I think it should be possible to use the std user client.